Doc/library/stdtypes.rst

```diff
@@ -479,6 +479,24 @@ class`. In addition, it provides a few more methods:
 
     .. versionadded:: 3.1
 
+.. method:: int.bit_count()
+
+    Return the number of ones in the binary representation of the integer,
+    excluding the sign bit::
```

Comment:
Suggested tweak to the wording, since we don't really have a sign "bit" as such in Python's long int representation.

```suggestion
    ignoring the sign::
```

Reply:
(Or maybe just "excluding the sign", to match the wording in `int.bit_length`.)

Reply:
Hmm. "The sign bit" refers to an implicit mental model that's open to confusion. I wouldn't mention "the sign bit" at all anywhere, but instead say, e.g.,

> Return the number of ones in the binary representation of the absolute value of the integer.

Then it doesn't matter whether the reader is thinking of 2's-comp, 1's-comp, signed-magnitude, or some insane 😉 way of representing negative ints.

Reply:
Update to use Tim's suggested wording.